11ty/definitions/r-word.md

```diff
@@ -0,0 +1,33 @@
+---
+title: R-word
+slug: r-word
+defined: true
+speech: noun
+flag:
+  level: avoid
+  text: ableist slur
+---
+
+Previously, the r-word was used in a medical context as an umbrella term to describe people with intellectual and developmental disabilities (I/DD).
```

Comment:
I think it's important to censor the word to prevent harm. I'm a little worried that excluding it many people might not know what word we mean? I'm curious about your thoughts on this.

Reply:
I think it's generally acceptable to use the full word when specifically discussing the usage of the word. Same with the complete phrase "mental retardation", which is used several times in the definition. I could use the full word, then the masking abbreviation in parentheses, and then refer to it as the "r-word" from then on? I could actually do the same thing with MR.

Reply:
I think that would be perfect! 

One of the items I'm opening an issue for is a broad-level content-warning at the top of the site.